templates/backup.yaml

```diff
@@ -444,7 +444,7 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       Handler: index.handler
-      Runtime: python3.7
+      Runtime: python3.11
```

Comment:
why are python3.11 only for this function?

Reply:
because only python3.11 supports the newest boto3 client for activation of the organization trusted access